pkg/server/service_windows.go

```diff
@@ -53,14 +55,16 @@ func (c *criService) getDefaultSnapshotterForSandbox(cfg *runtime.PodSandboxConf
 		platform string
 	)
 	if cfg != nil {
-		platform, _ = cfg.Labels["sandbox-platform"]
+		platform = strings.Replace(cfg.Labels["sandbox-platform"], "-", "/", -1)
```

Comment:
This change is urelated right? Why do we need to take it?

Reply:
I added two changes in this pr, as per the title. If you'd rather not have that change I can leave it out.

Reply:
oh wait, I guess this is a third change. it's need to let kubelet specify the label as it doesn't allow /

Reply:
This is a beaking change right now. Can we do this a formal change